hcl2template/types.packer_config.go

```diff
@@ -109,7 +109,8 @@ func (cfg *PackerConfig) EvalContext(ctx BlockContext, variables map[string]cty.
 			}),
 			buildAccessor: cty.UnknownVal(cty.EmptyObject),
 			packerAccessor: cty.ObjectVal(map[string]cty.Value{
-				"version": cty.StringVal(cfg.CorePackerVersionString),
+				"version":      cty.StringVal(cfg.CorePackerVersionString),
+				"iteration_id": cty.UnknownVal(cty.String),
```

Comment:
nit: We don't seem to use snake case naming for access variables. Should this be `iterationID` or `IterationID`?